leveldb_os/expiry_os_test.cc

```diff
@@ -1349,5 +1350,170 @@ TEST(ExpiryManifestTester, Compact1)
 }   // Compact1
 
 
+struct sExpiryDBObject
+{
+    const char * m_Key;   // string key
+    const char * m_Value; // string value
+    int m_NowMinus;       // expiry time to set
+};
+
+
+class ExpiryDBTester
+{
+public:
+    ExpiryDBTester()
+        : m_Good(false), m_DB(NULL),
+          m_BaseTime(port::TimeUint64())
+    {
+        m_DBName = test::TmpDir() + "/expiry";
+
+        // clean up previous execution
+        leveldb::DestroyDB(m_DBName, m_Options);
+
+        m_Options.create_if_missing=true;
+        m_Options.error_if_exists=false;
+        m_Expiry=new leveldb::ExpiryModuleOS;
```

Comment:
Per other discussions about the m_Expiry member, it would be good to add a comment that this data member is owned by the m_Options object (so no need to free in this class). That is, m_Expiry is simply a convenience pointer kept around to allow easier updating of the expiry policy in the test below. (This also applies to the m_Expiry data member in the ExpiryManifestTester class above.)